tests/browsercontext-route.spec.ts

```diff
@@ -235,3 +235,122 @@ it('should overwrite post body with empty string', async ({ context, server, pag
   const body = (await req.postBody).toString();
   expect(body).toBe('');
 });
+
+it.describe('service workers', () => {
+  it('intercepts initial script', async ({ server, page, context, browserName }) => {
+    it.fail(browserName === 'firefox');
```

Comment:
This works in webkit? Very surprising!

Reply:
Yes! WebKit this surprised me too! https://github.com/microsoft/playwright/runs/5453708442?check_suite_focus=true